code/modules/ninja/suit/ninjaDrainAct.dm

```diff
@@ -115,8 +115,8 @@ They *could* go in their appropriate files, but this is supposed to be modular
 			update_icon()
 
 /obj/machinery/proc/AI_notify_hack()
-	var/turf/location = get_turf(src)
-	var/alertstr = "<span class='userdanger'>Network Alert: Hacking attempt detected[location?" in [location]":". Unable to pinpoint location"]</span>."
+	var/area/location = get_area(src)
+	var/alertstr = "<span class='userdanger'>Network Alert: Hacking attempt detected[location?" in [location]":". Unable to pinpoint location"].</span>"
```

Comment:
var/alertstr = "<span class='userdanger'>Network Alert: Hacking attempt detected[get_area()?" in [get_area_name(src, TRUE)]":". Unable to pinpoint location"].</span>"

Reply:
and remove the cached var